samples/fhir-aad-b2c/oauth2Permissions.json

```diff
@@ -0,0 +1,280 @@
+{
+	"oauth2Permissions": [
+		{
```

Comment:
Can you delete this first JSON element in the array? We are not supporting patient.all.all.

```json
{
	"adminConsentDescription": "Allows user to read, write and delete all resources in a patient's compartment.",
	"adminConsentDisplayName": "patient.all.all",
	"id": "0d1a5f28-49cb-4a16-97f5-106efd659dc8",
	"isEnabled": true,
	"lang": null,
	"origin": "Application",
	"type": "User",
	"userConsentDescription": "Allows user to read, write and delete all resources in a patient's compartment.",
	"userConsentDisplayName": "patient.all.all",
	"value": "patient.all.all"
}
```

Reply:
Completed.